File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.XtraTab.Registrator;
using DevExpress.XtraTab.Drawing;
using DevExpress.XtraTab;
using System.Threading;

namespace WindowsApplication1
{
    public partial class Form1 : XtraForm
    {
        XtraTabProgressPage page = new XtraTabProgressPage();

        public Form1()
        {
            InitializeComponent();
            PaintStyleCollection.DefaultPaintStyles.Add(new MyRegistrator());
            xtraTabControl1.PaintStyleName = "MyStyle";
           
            page.Text = "Progress";
            page.ImageIndex = 0;
            xtraTabControl1.TabPages.Add(page);
        }

        private void simpleButton1_Click(object sender, EventArgs e)
        {
            for (int i = 0; i < 100; i++)
            {
                page.Progress = i;
                Thread.Sleep(20);
            }
        }
    }
}

File: VB/WindowsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.XtraTab.Registrator
Imports DevExpress.XtraTab.Drawing
Imports DevExpress.XtraTab
Imports System.Threading

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits XtraForm
		Private page As New XtraTabProgressPage()

		Public Sub New()
			InitializeComponent()
			PaintStyleCollection.DefaultPaintStyles.Add(New MyRegistrator())
			xtraTabControl1.PaintStyleName = "MyStyle"

			page.Text = "Progress"
			page.ImageIndex = 0
			xtraTabControl1.TabPages.Add(page)
		End Sub

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			For i As Integer = 0 To 99
				page.Progress = i
				Thread.Sleep(20)
			Next i
		End Sub
	End Class
End Namespace